README.md

```diff
@@ -45,29 +45,27 @@ The `student` is able to see, add and update their information, add, delete, and
 - Redux 
 - CSS
 - Socket.io
+- Google Firebase
 
 ## Technical Challenges
 
 - The largest technical challenge the team encounted was restructuring the entire application from `vanilla React` to React using `Next.js` 
 
-- One of the biggest learing points during this project was being able to look at legecy code, tear it down to the bare bones and rebuild it from the ground up with several unconnected components, then bringing them all together to make it mesh into a full functioning, fluid application. 
-
+- One of the biggest learing points during this project was being able to look at legecy code, and break down the changes that were previously made. Understand how everything was being called and breaking down the use of different state management [Redux]. Redux is a very powerful state managment tool when all parties involved know how to use the management system.  
 
 ## How does the *Hacking Transitions* application work?
 
 The application has a dedicated `Front-end` framework utilizing [React]() and [Next.js]() and managable state using [Redux](). All of the styling is done through `CSS`. This allows the user to make all the `CRUD` operation calls to the dedicated `Back-end` using [PSQL]().  
 
 The application utilizes [socket.io]() to conduct inner application chat operations. 
 
+The application uses Google Firebase as an authentication service to make the application more secure. 
+
 
 ## To Migrate/Seed Database
 Create your own .local.env as well as run the migration file 1st then the second run the seed.sql file. If you wish to see and test the Archive functionality then run the testArchive.sql file last.
 
 
-## App deployment on Vercel
-
-The easiest way to deploy a Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) 
-
 *Major issues*: does not support web sockets
 
 ## App deployment on Render
```

Comment:
Need to add in the render deployment part that since the application uses next.js it needs to be deployed as a web service since the FE and server are combined.